content/docs/reference/api/asm/stabilization/_index.md

```diff
@@ -683,3 +683,404 @@ aut contract call --address 0x29b2440db4A256B0c1E6d3B4CDcaA68E2440A08f underColl
+| `timeDue` | `uint` | The time the interest is due. The timestamp is provided as a [Unix time](/glossary/#unix-time) value |
+
+{{< alert title="Info" >}}
+For the default value set for `rate` see Reference, Genesis, [ASM stabilization config](/reference/genesis/#configasmstabilization-object).
+{{< /alert >}}
+  
+#### Response
+
+The function returns the amount of interest due as an `uint256` integer value.
+
+#### Event
+
+None.
+
+#### Usage
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="aut" >}}
+aut contract call --address 0x29b2440db4A256B0c1E6d3B4CDcaA68E2440A08f interestDue debt rate timeBorrow timeDue
+{{< /tab >}}
+{{< /tabpane >}}
+
+#### Example
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="aut" >}}
+aut contract call --address 0x29b2440db4A256B0c1E6d3B4CDcaA68E2440A08f interestDue 1000000000000000000 50000000000000000 1695308566 1697900566
+4118044981651418
+{{< /tab >}}
+{{< /tabpane >}}
+
+### underCollateralized
+
+Determines if a debt position is undercollateralized or not.
+
+Constraint checks are applied:
+
+- invalid price: the value of the `price` argument is valid, i.e. it is not equal to `0`.
+
+If the CDP is under collateralized, then it can be liquidated - see [`liquidate()`](/reference/api/asm/stabilization/#liquidate).
+
+{{< alert title="Info" >}}
+If a debt position is under collateralized or not is determined by calculating `(collateral * price) / debt`. If this returns a value `< liquidationRatio`, then the CDP is under collateralised and can be liquidated.
+{{< /alert >}}
+
+#### Parameters
+
+| Field | Datatype | Description |
+| --| --| --|
+| `collateral` | `uint256` | Amount of Collateral Token backing the debt |
+| `price` | `uint256` | The price of Collateral Token in Auton |
+| `debt` | `uint256` | The debt amount |
+| `liquidationRatio` | `uint256` | The liquidation ratio |
+
+{{< alert title="Info" >}}
+For the default value set for `liquidationRatio` see Reference, Genesis, [ASM stabilization config](/reference/genesis/#configasmstabilization-object).
+
+The current `price` value can be returned by calling [`collateralPrice()`](/reference/api/asm/stabilization/#collateralprice).
+{{< /alert >}}
+
+#### Response
+
+The method returns a boolean flag specifying whether the CDP is undercollateralized (true) or not (false).
+
+#### Event
+
+None.
+
+#### Usage
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="aut" >}}
+aut contract call --address 0x29b2440db4A256B0c1E6d3B4CDcaA68E2440A08f underCollateralized collateral price debt liquidationRatio
+{{< /tab >}}
+{{< /tabpane >}}
+
+#### Example
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="aut" >}}
+aut contract call --address 0x29b2440db4A256B0c1E6d3B4CDcaA68E2440A08f underCollateralized 206782464846980976 9672000000000000000 1000000000000000000 1800000000000000000
+false
+{{< /tab >}}
+{{< /tabpane >}}
```

Comment:
Yes, this is pre US spelling edit. That's the cause of the rebase dupe. Again, I removed this and it must have come back during rebase silently because there wasn't a conflict. I need to review my rebasing approach.